script/.ipynb_checkpoints/download_MNIST-checkpoint.py

```diff
@@ -0,0 +1,76 @@
+__author__ = "Xinqiang Ding <xqding@umich.edu>"
```

Comment:
this is also a checkpoint file, please delete and add `*.ipynb_checkpoints/` to your `.gitignore`

Reply:
deleted all checkpoints